haystack/nodes/reader/table.py

```diff
@@ -241,76 +243,68 @@ def __init__(
             model_name_or_path, revision=model_version, use_auth_token=use_auth_token
         )
         if tokenizer is None:
-            self.tokenizer = TapasTokenizer.from_pretrained(model_name_or_path, use_auth_token=use_auth_token)
+            self.tokenizer = TapasTokenizer.from_pretrained(
+                model_name_or_path, use_auth_token=use_auth_token, model_max_length=max_seq_len
+            )
         else:
-            self.tokenizer = TapasTokenizer.from_pretrained(tokenizer, use_auth_token=use_auth_token)
+            self.tokenizer = TapasTokenizer.from_pretrained(
+                tokenizer, use_auth_token=use_auth_token, model_max_length=max_seq_len
+            )
         self.max_seq_len = max_seq_len
         self.device = device
+        self.pipeline = _TableQuestionAnsweringPipeline(
+            task="table-question-answering",
+            model=self.model,
+            tokenizer=self.tokenizer,
+            framework="pt",
+            batch_size=1,  # batch_size of 1 only works currently b/c of issue with HuggingFace pipeline logic and the return type of TableQuestionAnsweringPipeline._forward
```

Comment:
Should we expose this information to the user as a log line? Do you think it's good for users to know this?

Reply:
I don't think this is necessary for users to know. This is still the same behavior as before and the predict and predict_batch methods still work.

Reply:
As in there was not a working batch_size parameter for this TableReader. I think once we add support for it would be good to advertise as a new feature.